src/locale/locales/fr/messages.po

```diff
@@ -5275,12 +5255,12 @@ msgstr "Connecté en tant que @{0}"
 
 #: src/view/com/notifications/FeedItem.tsx:197
 msgid "signed up with your starter pack"
-msgstr ""
+msgstr "s’est inscrit·e avec votre pack de démarrage"
```

Comment:
I concur… I can't find a better way to write this using _langage épicène_, so _point médian_ is the way.